clients/csrf/csrf_token_fetcher_impl.go

```diff
@@ -0,0 +1,72 @@
+package csrf
+
+import (
+	"github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/csrf/csrf_paramters"
+	"github.com/cloudfoundry/cli/plugin"
+	"net/http"
+	"os"
+)
+
+const CsrfTokenHeaderFetchValue = "Fetch"
+const CsrfTokensApi = "/api/v1/csrf-token"
+const ContentTypeHeader = "Content-Type"
+const AuthorizationHeader = "Authorization"
+const ApplicationJsonContentType = "application/json"
+const CookieHeader = "CookieHeader"
+
+type CsrfTokenFetcherImpl struct {
```

Comment:
*Impl is not a good naming, IMO. Maybe DefaultCsrfTokenFetcher?

Reply:
Renamed to -> DefaultCsrfTokenFetcher